apps/designer/app/designer/features/style-panel/shared/property-name.tsx

```diff
@@ -1,39 +1,89 @@
-import { Flex, Label, Tooltip } from "@webstudio-is/design-system";
+import { useState } from "react";
+import type { StyleProperty } from "@webstudio-is/css-data";
+import {
+  Button,
+  Flex,
+  Box,
+  Text,
+  Label,
+  Tooltip,
+  Popover,
+  PopoverContent,
+  PopoverPortal,
+  PopoverTrigger,
+  Separator,
+} from "@webstudio-is/design-system";
+import { UndoIcon } from "@webstudio-is/icons";
 import { useIsFromCurrentBreakpoint } from "./use-is-from-current-breakpoint";
-import { propertyNameColorForSelectedBreakpoint } from "./constants";
-import type { PropertyProps } from "../style-sections";
 
-export const PropertyName = ({ property, label, css }: PropertyProps) => {
+type PropertyProps = {
+  property: StyleProperty | StyleProperty[];
+  label: string;
+  onReset: () => void;
+};
+
+export const PropertyName = ({ property, label, onReset }: PropertyProps) => {
   const isCurrentBreakpoint = useIsFromCurrentBreakpoint(property);
+  const [isOpen, setIsOpen] = useState(false);
+  const isPopoverEnabled = isCurrentBreakpoint;
+
+  const labelElement = (
+    <Label
+      css={{
+        fontWeight: "inherit",
+        padding: "calc($spacing$3 / 2) $spacing$3",
+        ...(isCurrentBreakpoint
+          ? {
+              color: "$blue11",
+              backgroundColor: "$colors$blue4",
+              borderRadius: "$borderRadius$4",
+            }
+          : {
+              color: "$hiContrast",
+            }),
+      }}
+      htmlFor={property.toString()}
+    >
+      {label}
+    </Label>
+  );
+
+  if (isPopoverEnabled) {
+    return (
+      <Flex align="center">
+        <Popover modal open={isOpen} onOpenChange={setIsOpen}>
+          <PopoverTrigger asChild aria-label="Show proprety description">
+            {labelElement}
+          </PopoverTrigger>
+          <PopoverPortal>
+            <PopoverContent align="start" onClick={() => setIsOpen(false)}>
+              <Flex align="start" css={{ padding: "$spacing$3 $spacing$4" }}>
```

Comment:
nit: we should use less shorthand properties and more longhand or something more practical for this case: `px: '$spacing$4', py: '$spacing$3'`